src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        AreaCalculator [] figures = new AreaCalculator[6];
```

Comment:
Why are you using `AreaCalculator` type instead of `AbstractFigure`? It's more general and will allow you to call methods without casting.